xla/python/py_client_gpu.cc

```diff
@@ -173,4 +179,17 @@ XLA_REGISTER_CUSTOM_CALL_TARGET_WITH_SYM(
     "xla_python_gpu_callback", &XlaPythonGpuCallback,
     absl::AsciiStrToUpper(PlatformUtil::CanonicalPlatformName("gpu").value()));
 
+absl::Status JaxLog(absl::string_view prefix, ffi::RemainingArgs args) {
```

Comment:
Question: How do we handle different types in FFI?

Let's say I am going to print for F32, BF16, etc. Do we need some template metaprogramming here?